### ISSUE TITLE
[Feature]: AutoE2E Publication, Benchmarking and Code Cleanup

### ISSUE THREAD

**Issue by m-zain-khawaja** [autowarefoundation/auto_e2e#210]
### Problem / motivation

We have published our open-loop model artefacts on HuggingFace, we need to finalize our release of the models, and benchmarks - link: https://huggingface.co/AutowareFoundation/auto_e2e

We need to finalize publication of HuggingFace models with their appropriate benchmarks and visualizations as a formal Autoware Robotaxi WG release and we also need to write a paper which summarises our work.

At the same time, our repo has become quite bloated with un-used code and tests, which we should clean-up.

## 1 - Visualizations
We need visualizations of the NuPlan trained model on the NuPlan dataset

## 2 - Benchmarks
We have benchmarks for ADE and FDE for our baseline NuPlan trained model and KITScenes finetuned model only on the KITScenes Val and Test sets.

### We should benchmark our model on other public datasets:

We should benchmark other open models, e.g. Alpamayo 1.5, SimForge, DriveJEPA, UniAD, METEOR on KITScenes dataset and NuPlan dataset so that we can have an apples-to-apples comparison.

**KIT Scenes Test Set (V3.5) - ADE/FDE metrics**
AutoE2E NuPlan, AutoE2E KIT, Alapamayo 1.5, SimForge, DriveJEPA, UniAD, METEOR, VAD, Qwen-Drive-1.0

**NuPlan Test Set - ADE/FDE metrics**
AutoE2E NuPlan, AutoE2E KIT, Alapamayo 1.5, SimForge, DriveJEPA, UniAD, METEOR, VAD, Qwen-Drive-1.0

**Metrics to be reported:**
ADE/FDE at 1s, 2s, 3s, 4s and 5s

**Request to code contributors** - please start the benchmarking and report in the comments what model you are benchmarking and on what dataset and then share your results in the comments. Please READ the comments so you don't double up on a benchmark.

**Pipeline for testing/benchmarking is provided, use as is**

**Official public benchmarking of AutoE2E KITScenes Epoch 5 model**
Please submit the AutoE2E KITScenes Epoch 5 model checkpoint trajectory outputs to the KITScenes lab for official evaluation and public benchmarking release

## 3 - Code cleanup
After 1) and 2) are completed, we need to purge all unnecessary tests and features and parts of the codebase which are not leveraged and update the documentation. We should remove all unnecessary feature branches and merge the core features into the main branch.

## 4 - Paper publication
We need to summarize our findings as an academic paper and publish on ArXiv. 

## 5 - HuggingFace publication
We need to fix and clean up the HuggingFace publication and make it easy for users to get started with the model

**Comment by Nishad-Sahu** [comment]
Hi, as discussed in the E2E working group meeting on 23rd Sep, I can try running the Benchmarking for UniAD.

**Comment by Arseni10Lk** [comment]
Hey, I am benchmarking AutoE2E NuPlan on KITScenes test set 3.5
